packages/core-data/src/entities.js

```diff
@@ -189,6 +189,7 @@ function* loadPostTypeEntities() {
 				record?.title ||
 				( isTemplate ? startCase( record.slug ) : String( record.id ) ),
 			__unstablePrePersist: isTemplate ? undefined : prePersistPostType,
+			__unstable_rest_base: postType.rest_base,
```

Comment:
Ideally this should be something that is available in all entities (and not just post one) but also ideally, it could potentially be better though, basically we need this properly to match with the "permissions" array, we could have a `canEdit` function per entity or a `getResource` or something like that. But this will do for now.